taichi_3d_gaussian_splatting/GaussianPointCloudRasterisation.py

```diff
@@ -334,6 +320,15 @@ def generate_point_attributes_in_camera_plane(  # from 3d gaussian to 2d feature
 def gaussian_point_rasterisation(
     camera_height: ti.i32,
     camera_width: ti.i32,
+    q_camera_pointcloud: ti.types.ndarray(ti.f32, ndim=2),  # (K, 4)
+    t_camera_pointcloud: ti.types.ndarray(ti.f32, ndim=2),  # (K, 3)
+    point_object_id: ti.types.ndarray(ti.i32, ndim=1),
+    # (M) the point id in the view frustum.
+    point_id_list: ti.types.ndarray(ti.i32, ndim=1),
+    camera_intrinsics: ti.types.ndarray(ti.f32, ndim=2),  # (3, 3)
+    pointcloud: ti.types.ndarray(ti.f32, ndim=2),  # (N, 3)
+    # (N, 56) 56 features (cov_rotation, xxx, r, g, b)
```

Comment:
Not 56 features after using the MLP. Maybe change to:
#(N, feature_num), feature_num features (cov_rotation, ..., mlp weights and bias)